crates/biome_cli/src/execute/migrate/eslint_eslint.rs

```diff
@@ -364,10 +364,8 @@ impl<T: Deserializable + 'static, U: Deserializable + 'static> Deserializable fo
                 Some(RuleConf::Spread(severity, spread))
             }
         }
-        if matches!(
-            value.visitable_type()?,
-            VisitableType::NUMBER | VisitableType::STR
-        ) {
+        let visitable_type = value.visitable_type()?;
+        if visitable_type == VisitableType::NUMBER || visitable_type == VisitableType::STR {
```

Comment:
This if condition doesn't look equivalent to the previous version to me.

Reply:
I'm not 100% sure, but I think 
```
VisitableType::NUMBER | VisitableType::STR
----------------------^ this `|` this a part of pattern not a bit or operator of `VisitableType`.
```
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=f4532bf01ca5080a6e6de0e38b7ff1ac

btw compiler don't allow me to use matches! with BitFlag
```
error: to use a constant of type `enumflags2::BitFlags<biome_deserialize::diagnostics::Types, u8>` in a pattern, `enumflags2::BitFlags<biome_deserialize::diagnostics::Types, u8>` must be annotated with `#[derive(PartialEq)]`
   --> crates/biome_cli/src/execute/migrate/eslint_eslint.rs:369:13
    |
369 |             VisitableType::NUMBER | VisitableType::STR
    |             ^^^^^^^^^^^^^^^^^^^^^
    |
    = note: the traits must be derived, manual `impl`s are not sufficient
    = note: see https://doc.rust-lang.org/stable/std/marker/trait.StructuralPartialEq.html for details

```

Reply:
Oh, yeah you're totally right. Nevermind me then.